packages/workbox-strategies/src/Strategy.ts

```diff
@@ -115,7 +110,7 @@ abstract class Strategy implements RouteHandlerObject {
    * @param {URL} [options.url]
    * @param {*} [options.params]
    */
-  handle(options: FetchEvent | RouteHandlerCallbackOptions): Promise<Response> {
+  handle(options: HandlerCallbackOptions): Promise<Response> {
```

Comment:
I think I'd still like to have this to accept a `FetchEvent` object, since that's how it's most often going to be used:

```js
addEventListener('fetch', (event) => {
   event.respondWith(someStrategy.handle(event));
});
```

I remember in the past you expressed a preference for `handle({event})` (using object shorthand), but that doesn't work if we're going to require both the `request` and the `event` param (which I think we want to?).

WDYT?

Reply:
Yeah, I'm adding that `FetchEvent` back as one of the supported types, especially since the code supports it.

Reply:
Cool.